author.php

```diff
@@ -85,46 +81,20 @@
 					<?php if ( have_posts() ) : ?>
 						<?php while ( have_posts() ) : the_post(); ?>
 
-                            <?php if (strcmp(get_post_status(), "publish") == 0) :?>
-
-                                <li class="list-group-item">
-
-                                    <div class="row">
-                                        <div class="col-3 align-self-center">
-                                            <a href="<?php the_permalink() ?>">
-                                                <?php if (has_post_thumbnail()) : ?>
-                                                    <?php the_post_thumbnail('full', array( 'class' => 'img-thumbnail' )); ?>
-                                                <?php endif; ?>
-
-
-
+                            <?php
```

Comment:
I hate to remove so much of your code, but I think it's better to reuse as much code/styles as possible to keep things consistent. Sorry, I should've taken a better look at this file and suggested this earlier.